### PR TITLE
PADV-1472 frontend - Create lab details Json section 3rd

### PR DESCRIPTION
**Pull request by Nekenhei**
## Tickets

- https://agile-jira.pearson.com/browse/PADV-1472

## Description
This PR adds a feature to display the entire JSON response of the /details endpoint in a viewer.

## Changes Made

- [x] Add dependency of the jsoneditor JS library
- [x] Add component JSON viewer to the details view 

## How To Test

Before any prior step, go to your MFE folder and execute `npm i` to install the new dependencies.

Please have your devstack environment running, run `npm start` and the MFE shall be running in port 2003 then, please go to the LMS and access to a skillable ready course, there into the Skillable Tab, the MFe will be shown, access any student of the roster, select one of the given laboratories and the view shall be displayed

image

### CODE REVIEW

File: src/components/JsonViewer/index.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import JSONEditor from 'jsoneditor';
+import 'jsoneditor/dist/jsoneditor.css';
+
+const JsonViewer = ({ labName, labData }) => {
+  const editorRef = useRef(null);
+  const containerRef = useRef(null);
+
+  useEffect(() => {
+    const options = {
+      mode: 'view', // set mode to 'view' to disable editing
```
Comment (01001110J) [inline src/components/JsonViewer/index.jsx:20]: I think this comment is a little bit confusing, what about:

```suggestion
      mode: 'view', // Disable editing
```

File: src/components/LabDetails/index.jsx [hunk 2]
```diff
@@ -88,24 +89,31 @@ const LabDetails = ({
...
+                state: labDetails.State,
+                completionStatus: labDetails.CompletionStatus,
+                totalRunTime: labDetails.TotalRunTime,
+                examPassed: labDetails.exam_passed ? 'Yes' : 'No',
+              }}
+            />
+          </div>
+          <div className="col-12 col-md-5">
+            {/* Useful for further features. */}
+          </div>
```
Comment (01001110J) [inline src/components/LabDetails/index.jsx:111]: What's the point of keep this?

File: src/components/JsonViewer/index.jsx
Comment (01001110J) [inline src/components/JsonViewer/index.jsx:1]: @Nekenhei Is possible to change that blue color? Maybe using a ccs class or something, also I suggest to add more margin on the top to have more space between this secitons

image

File: src/components/LabDetails/index.jsx (hunk 2 above)
Comment (nandodev-net) [reply src/components/LabDetails/index.jsx:111]: @01001110J Here will be the card with the pie charts

Comment (Nekenhei) [reply src/components/LabDetails/index.jsx:111]: I was, without any doubt, sure that without this, the card won't be shown well... but your right, it doesn't needed.

File: src/components/JsonViewer/index.jsx
Comment (nandodev-net) [reply src/components/JsonViewer/index.jsx:1]: @Nekenhei Try this 

image

**Review by nandodev-net — APPROVED** [review]

File: src/components/JsonViewer/index.jsx
Comment (Nekenhei) [reply src/components/JsonViewer/index.jsx:1]: image

I've just set the same color as the hyperlink in breadcrumb

**Review by 01001110J — APPROVED** [review]

**Review by JuanDavidBuitrago — APPROVED** [review]

**Pull request merged**